sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/SelectionMarkState.java

```diff
@@ -9,6 +9,13 @@
  * Defines values for SelectionMarkState. i.e., Selected or Unselected.
  */
 public final class SelectionMarkState extends ExpandableStringEnum<SelectionMarkState> {
+
+    /**
+     * Constructs a SelectionMarkState object.
+     */
+    public SelectionMarkState() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.